src/Cake.Common/Tools/GitReleaseManager/AddAssets/GitReleaseManagerAssetsAdder.cs

```diff
@@ -152,12 +151,14 @@ private ProcessArgumentBuilder GetArguments(string token, string owner, string r
             builder.Append("--token");
             builder.AppendQuoted(token);
```

Comment:
Secret

Reply:
Yes, that is a great catch!  Not sure how/why this wasn't like this already?!?